jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.form

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="software.aws.toolkits.jetbrains.services.sqs.toolwindow.SendMessagePane">
-  <grid id="27dc6" binding="component" layout-manager="GridLayoutManager" row-count="2" column-count="8" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
-    <margin top="10" left="10" bottom="10" right="10"/>
+  <grid id="27dc6" binding="component" layout-manager="GridLayoutManager" row-count="2" column-count="5" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```

Comment:
Can a screenshot be attached for the UI changes? Also, if the margin values are all 0s and assuming that's the default, maybe we can remove the tag unless they are generated along with the form?

Reply:
It is generated by the GUI editor.

Screenshot:
![Screen Shot 2021-06-29 at 9 39 57 AM](https://user-images.githubusercontent.com/8992246/123835940-fc679f00-d8bd-11eb-8685-bddea8709fd6.png)